File: occupancy-manager-java/src/main/java/com/occupancy/manager/dto/OccupancyUsageResponse.java
package com.occupancy.manager.dto;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

/**
*
* @author Ramachandra
*
*/

@Setter
@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class OccupancyUsageResponse {

	@JsonProperty("Usage Premium")
	private String premiumUsage;
	@JsonProperty("Usage Economy")
	private String economyUsage;
}

File: occupancy-manager-kotlin/src/main/kotlin/com/occupancy/manager/dto/OccupancyUsageResponse.kt
package com.occupancy.manager.dto

import com.fasterxml.jackson.annotation.JsonProperty

/**
 *
 * @author Ramachandra
 */
data class OccupancyUsageResponse(
	@JsonProperty("Usage Premium") var premiumUsage: String,
	@JsonProperty("Usage Economy") var economyUsage: String,
)